docs/guides/tutorials/getting_started/connect_to_data.rst

```diff
@@ -26,25 +26,34 @@ Let's configure your first Datasource, a connection to the local Postgres databa
     : 2
     
     Give your new Datasource a short name.
-     [my_postgres_db]:
+     [my_postgres_db]: <hit enter>
     
     Next, we will configure database credentials and store them in the `my_postgres_db`
     section of this config file: great_expectations/uncommitted/config_variables.yml:
 
-    Would you like to proceed? [Y/n]:
+    Would you like to proceed? [Y/n]: <hit enter>
```

Comment:
This is pedantic, but maybe "press" instead of "hit" (it sounds more gentle to me)...